patches/minecraft/net/minecraft/item/ItemBlock.java.patch

```diff
@@ -53,4 +53,10 @@
 +
 +        return true;
 +    }
++
++    @SideOnly(Side.CLIENT)
++    public void func_77624_a(ItemStack stack, EntityPlayer playerIn, List<String> tooltip, boolean advanced)
++    {
++        field_150939_a.addInformation(stack, playerIn, tooltip, advanced);
```

Comment:
It should be `this.field_150939_a.addInformation(stack,` playerIn, tooltip, advanced);` ??